src/templates/mdx_template.js

```diff
@@ -11,12 +11,16 @@ import SEO from "components/seo";
 import Content from "components/ContentBanner";
 import HyperLink from "components/highLightLink";
 import PrevNextNav from "components/prevNextNav";
+// eslint-disable-next-line no-unused-vars
+import MCQ from "components/mcq";
+//import Truefalse from "components/truefalse";
```

Comment:
I don't know if commented out imports are really serving a purpose here

Reply:
They are not I shall remove them